lib/chef_apply/dist.rb

```diff
@@ -0,0 +1,80 @@
+module ChefApply
+  module Dist
+    # This class is not fully implemented, depending on it is not recommended!
+    # When referencing a product directly, like Chef (Now Chef Infra)
+    PRODUCT = "Chef Infra Client".freeze
+
+    # The name of the server product
+    SERVER_PRODUCT = "Chef Infra Server".freeze
+
+    # Short name for Chef Infra
+    SHORT = "Chef".freeze
+
+    # The client's alias (chef-client)
+    CLIENT = "chef-client".freeze
+
+    # name of the automate product
+    AUTOMATE = "Chef Automate".freeze
+
+    # The chef executable, as in `chef gem install` or `chef generate cookbook`
+    EXEC = "chef".freeze
+
+    # The workstation's product name
+    WORKSTATION = "Chef Workstation".freeze
+
+    # product website address
+    WEBSITE = "https://chef.io".freeze
+
+    # chef-apply's product name
+    APPLY = "Chef Infra Apply".freeze
```

Comment:
Hmm, hadn't thought about this one. Seems right to me, but I could be persuaded that `Chef Apply` is better. I think `Chef Infra Apply` is more appropriate since it is a subset of the `chef` command. @chef/chef-workstation-owners what do y'all think?

Reply:
I would personally check with marketing/docs teams. I vote as is `Chef Infra Apply`. 🥇

Reply:
I've sent the question over to marketing and will post back with any updates.

Reply:
Any news on this? I'm going through workstation dependencies to update Cinc's backlog.

Reply:
This turns out to be a little tricky, because 'chef-apply' (chef-run) isn't a product itself. We're continuing to have discussions around this internally, but I don't yet have a time frame for when we'll have a definitive answer. 

It looks like we're not referencing this in the chef_apply repository - would it make sense to remove this const for now, or would that cause problems for Cinc's implementation?

Reply:
If it's not a word we need to worry about then it could certainly be removed. I'll try to book some time to fix up this PR over the next two weeks